loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowProviderPluginInfo.java

```diff
@@ -13,9 +13,9 @@
  * @author Mathieu Bague <mathieu.bague at rte-france.com>
  */
 @AutoService(PluginInfo.class)
-public class LoadFlowFactoryPluginInfo extends PluginInfo<LoadFlowFactory> {
+public class LoadFlowProviderPluginInfo extends PluginInfo<LoadFlowProvider> {
```

Comment:
Don't forget to update this [page](http://www.powsybl.org/docs/tools/plugins-info.html)

Reply:
Done, added load-flow.md